docs/Glossary.md

```diff
@@ -0,0 +1,10 @@
+# Glossary
+
```

Comment:
Could you add some sort of description to the glossary? some foreword or something

after that I'm good to merge it in in this state